pkg/controller/podautoscaler/horizontal.go

```diff
@@ -139,141 +150,140 @@ func (a *HorizontalController) Run(stopCh <-chan struct{}) {
 	glog.Infof("Shutting down HPA Controller")
 }
 
-// getLastScaleTime returns the hpa's last scale time or the hpa's creation time if the last scale time is nil.
-func getLastScaleTime(hpa *autoscaling.HorizontalPodAutoscaler) time.Time {
```

Comment:
Ah, I know why this got removed -- it's largely irrelevent with the improved handling for missing pods -- under the new scheme, we'll only error out if no metrics *at all* are present, which will only occur a) right after the deployment is initially created, or b) if we have an actual error.